python/ql/test/2/library-tests/PointsTo/import_time/Pruned.ql

```diff
@@ -7,5 +7,5 @@ from ControlFlowNode f, Location l, Context c
 
 where not PointsToInternal::reachableBlock(f.getBasicBlock(), c) and c.isImport() and
 (f.getNode() instanceof FunctionExpr or f.getNode() instanceof ClassExpr) and
-l = f.getLocation() and l.getFile().getName().matches("%test.py")
-select l.getStartLine()
\ No newline at end of file
+l = f.getLocation() and l.getFile().getShortName() = "test.py"
```

Comment:
This rewrite is more specific, as it doesn't accept e.g. files named `footest.py`.

Reply:
I guess this might be okay, since the tests still pass.

Reply:
I looked at the test setup, and in the cases with only one test file (`test.py`), I changed it this way.